DESCRIPTION

```diff
@@ -1,60 +1,65 @@
 Type: Package
 Package: testdat
 Title: Data Unit Testing for R
-Version: 0.2.0
+Version: 0.2.0.9000
 Authors@R: 
     c(person(given = "Danny",
              family = "Smith",
              role = c("aut", "cre"),
-             email = "daniel.smith@srcentre.com.au"),
+             email = "danny@gorcha.org"),
       person(given = "Kinto",
              family = "Behr",
-             role = c("aut"),
+             role = "aut",
              email = "kinto.behr@srcentre.com.au"),
       person(given = "The Social Research Centre",
              role = "cph"))
-Description: An extension of 'testthat' made for checking data
-    frames.
+Description: Test your data! An extension of the 'testthat' unit testing
+    framework with a family of functions and reporting tools for checking
+    and validating data frames.
 License: MIT + file LICENSE
+URL: https://socialresearchcentre.github.io/testdat/,
+    https://github.com/socialresearchcentre/testdat
+BugReports: https://github.com/socialresearchcentre/testdat/issues
 Depends:
     R (>= 3.2.2),
     testthat (>= 2.0.0)
 Imports:
     dplyr (>= 0.8.0),
     glue,
```

Comment:
Do we need both stringr and glue?

Reply:
I prefer to use `glue()` rather than `str_glue()` - the latter is a wrapper for the former so it's an indirect glue dependency anyway.